.circleci/config.yml

```diff
@@ -31,23 +31,29 @@ references:
         coverage report
         COVERALLS_REPO_TOKEN=QEAuo5Pufwd8TGqQiqt7a1HWWp3i9mL0Y coveralls
 
-jobs:
-
-  test_archlinux:
-    docker:
-      - image: iterait/archlinux
+  test_job: &test_job
     working_directory: ~/apistrap
     steps:
-      - setup_remote_docker:
-          docker_layer_caching: true
-          version: 17.11.0-ce
       - checkout
       - *install
       - *test
 
+
```

Comment:
I do not feel like we need to test both 3.7 and 3.8.

Reply:
I do though. 3.8 will take some time to become mainstream due to ABI incompatibilities, but we sure want to support it. In the meantime, supporting 3.7 is also necessary. Also, it's basically free.